src/log/context.cr

```diff
@@ -51,5 +51,11 @@ end
 
 class Fiber
   # :nodoc:
-  property logging_context : Log::Context = Log::Context.new
+  getter logging_context : Log::Context = Log::Context.new
+
+  # :nodoc:
+  def logging_context=(value : Log::Context)
+    raise ArgumentError.new "Expected hash context, not #{value}" unless value.as_h?
```

Comment:
```suggestion
    raise ArgumentError.new "Expected hash context, not #{value.class}" unless value.as_h?
```

Reply:
Is there anything wrong with showing the value in the exception?

Reply:
Showing only the type is more concise and I don't think we need the value here.
It would also be confusing when `value.to_s` looks like a stringified hash.